installer/charts/values.yaml.tpl

```diff
@@ -9,7 +9,7 @@
 {{- $ingressDomain := required "OpenShift ingress domain" .OpenShift.Ingress.Domain -}}
 {{- $ingressRouterCA := required "OpenShift RouterCA" .OpenShift.Ingress.RouterCA -}}
 {{- $openshiftMinorVersion := required "OpenShift Version" .OpenShift.MinorVersion -}}
-{{- $keycloakEnabled := or $tpa.Enabled $tas.Enabled }}
+{{- $keycloakEnabled := or $tpa.Enabled $tas.Enabled $rhdh.Enabled }}
```

Comment:
That's a bit more complicated no? As we need RHDH and the OIDC provider if I am not mistaken.